src/platform/plugins/shared/dashboard/common/content_management/v1/types.ts

```diff
@@ -13,7 +13,7 @@ import type {
   SavedObjectUpdateOptions,
 } from '@kbn/content-management-utils';
 import { Serializable } from '@kbn/utility-types';
-import { RefreshInterval } from '@kbn/data-plugin/common';
```

Comment:
Should `@kbn/data-plugin/common` re-export `@kbn/data-service-server` to avoid having to update all imports?

Reply:
It is just a type, so I don't think it matters for bundle size if the consumer is already using items from the data plugin. I'll update this PR to re-export the type from `@kbn/data-plugin/common` for convenience. And we probably won't need reviews from other teams.